lib/cmd/fhc/target.js

```diff
@@ -137,7 +137,7 @@ function undefinedTargCleanup(cb, tar) {
 }
 
 function pingTarget(targ, callback) {
-  request(targ + "/box/srv/1.1/act/sys/auth/logout", function(err, response, body){
```

Comment:
@pb82 FYI

I think we should change this anyway, but I was seeing a bad request error which spurred on the fix. Might be related to the CSRF issues with old EP's we've been seeing over recent days.